Cargo.toml

```diff
@@ -21,11 +21,11 @@ license = "Apache-2.0"
 [workspace.dependencies]
 candid = "0.10.4"
 candid_parser = "0.1.4"
-ic-agent = { git = "https://github.com/dfinity/agent-rs.git", rev = "7869e06b048a69210849bd43ed939fe0c67df564" }
+ic-agent = { git = "https://github.com/dfinity/agent-rs.git", rev = "5b8bf70871e23c53ee3bfe6d60911ecfa48abe48" }
```

Comment:
When approved please remember to update these to point to a main branch commit before merging